packages/admin-panel/src/VizBuilderApp/api/mutations/useExportVisualisation.js

```diff
@@ -0,0 +1,18 @@
+/**
+ * Tupaia
+ * Copyright (c) 2017 - 2021 Beyond Essential Systems Pty Ltd
+ */
+
+import { useMutation } from 'react-query';
+import { download } from '../api';
+
+export const useExportVisualisation = config =>
+  useMutation(['export/dashboardVisualisation', config], () =>
```

Comment:
This method is called `useExportVisualisation`, but specifically calls `export/dashboardVisualisation`. We can refactor this when we add map overlay support, either by providing the type of viz to export (if the process is almost identical), or by renaming it to `useExportDashboardVisualisation`.

The decision was made based on an existing mutation (`useSaveVisualisation`)

Reply:
Yep I agree, for front end, I'm leaning towards the generic term `visualisation` if possible (because I think mostly the front end process between building map overlay and building dashboard are the same, and they will most likely use the same visualisation schema, ie: (having `data`, `presentation`,...). The only difference is just calling different endpoints based on the type. It's still very early to tell but that's my gut feeling!

Only when it comes to the back end, then I'm leaning towards calling them separately (ie: DashboardVisualisation, MapOverlayVisualisation,...). This is because I think backend process for map overlays and dashboards are likely going to be different.